slowfast/models/video_model_builder.py

```diff
@@ -7,6 +7,8 @@
 import torch
 import torch.nn as nn
 
+from fairscale.nn.checkpoint import checkpoint_wrapper
```

Comment:
Can we add a try except loop if user does not have the package?

Reply:
great idea. done.